python/ray/serve/_private/deploy_utils.py

```diff
@@ -122,6 +121,7 @@ def deploy_args_to_deployment_info(
         app_name=app_name,
         route_prefix=route_prefix,
         docs_path=docs_path,
+        ingress=ingress,
```

Comment:
is this a separate bugfix?

Reply:
It's not a bugfix, but with the small restructure in this PR, I decided to include `ingress` boolean in deployment infos so it's easier to unify the codepath between
* python serve.run() deploy
* heavyweight config deploy with kicking off build task
* lightweight config deploy that only applies override options